src/app/shared/components/catalog/catalog.component.html

```diff
@@ -0,0 +1,23 @@
+<h1 class="menu-title">Menu items</h1>
+<div class="container">
+  <div *ngFor="let item of items" class="card-container">
+    <h3 class="card-price">${{ item.price }}</h3>
+    <h2 class="card-title">{{ item.title }}</h2>
+    <figure class="card-image-container">
+      <img [src]="item.imageUrl" alt="product-image" />
+    </figure>
+    <div class="card-buttons">
+      <button
+        class="btn btn--edit"
+        routerLink="edit/:{{ item.id }}"
```

Comment:
`:` is not needed. It's only needed for route definitions.
```suggestion
        [routerLink]="['edit', item.id]"
```